backend/server.js

```diff
@@ -36,76 +37,169 @@ app.use(
+      field_goals: player.field_goals,
+      field_percent: field_percent,
+      ft: player.ft,
+      ft_percent: ft_percent,
+      fta: player.fta,
+      games: player.games,
+      games_started: player.games_started,
+      id: player.id,
+      minutes_played: player.minutes_played,
+      player_name: player.player_name,
+      team: player.team,
+      three_attempts: player.three_attempts,
+      three_fg: player.three_fg,
+      three_percent: three_percent,
+      two_attempts: player.two_attempts,
+      two_fg: player.two_fg,
+      two_percent: two_percent,
+    },
+  });
+}
+
+async function changePlayerData(player, oldPlayer) {
+  if (!isEqual(player, oldPlayer)) {
+    const field_percent =
+      player.field_attempts === 0
+        ? 0
+        : player.field_goals / player.field_attempts;
+    const ft_percent = player.fta === 0 ? 0 : player.ft / player.fta;
+    const three_percent =
+      player.three_attempts === 0 ? 0 : player.three_fg / player.three_attempts;
+    const two_percent =
+      player.two_attempts === 0 ? 0 : player.two_fg / player.two_attempts;
+    const effect_fg_percent = !player.effect_fg_percent
+      ? "0"
+      : player.effect_fg_percent;
+
+    await prisma.player.update({
+      where: {
+        id: oldPlayer.id,
+      },
+      data: {
+        AST: player.BLK,
+        DRB: player.DRB,
+        ORB: player.ORB,
+        PF: player.PF,
+        PTS: player.PTS,
+        STL: player.STL,
+        TOV: player.TOV,
+        TRB: player.TRB,
+        BLK: player.BLK,
+        age: player.age,
+        effect_fg_percent: effect_fg_percent,
+        field_attempts: player.field_attempts,
+        field_goals: player.field_goals,
+        field_percent: field_percent,
+        ft: player.ft,
+        ft_percent: ft_percent,
+        fta: player.fta,
+        games: player.games,
+        games_started: player.games_started,
+        id: player.id,
+        minutes_played: player.minutes_played,
+        player_name: player.player_name,
+        team: player.team,
+        three_attempts: player.three_attempts,
+        three_fg: player.three_fg,
+        three_percent: three_percent,
+        two_attempts: player.two_attempts,
+        two_fg: player.two_fg,
+        two_percent: two_percent,
+      },
+    });
+  }
+}
+
+function isEqual(player, oldPlayer) {
+  return (
+    player.player_name === oldPlayer.player_name &&
+    player.age === oldPlayer.age &&
+    player.minutes_played === oldPlayer.minutes_played
+  );
+}
+
+async function run() {
+  let queryUrl = new URL(
+    "https://nba-stats-db.herokuapp.com/api/playerdata/season/2023"
+  );
+  do {
+    const response = await fetch(queryUrl);
+    const data = await response.json();
+    const players = data.results;
+    queryUrl = data.next;
+
+    for (let i = 0; i < players.length; ++i) {
+      updatePlayer(players[i]);
+    }
+  } while (queryUrl !== null);
+}
+
+cron.schedule("0 1 * * *", function () {
+  run();
```

Comment:
This will update the database with the data as shown by the sample
<img width="1612" alt="Screenshot 2024-07-02 at 7 33 10 PM" src="https://github.com/GloriaXM/numbers-game/assets/112780860/3c6f950a-79e5-4bb5-9a0c-255f353f4257">

Reply:
Old data will be left untouched until the data has changed as shown below:
<img width="1619" alt="Screenshot 2024-07-02 at 7 37 04 PM" src="https://github.com/GloriaXM/numbers-game/assets/112780860/96f27d84-0b8a-406a-b0ac-27e5575ed90c">